src/mrpro/data/KHeader.py

```diff
@@ -270,6 +274,180 @@ def from_ismrmrd(
             ) from None
         return instance
 
+    def to_ismrmrd(self) -> ismrmrdschema.ismrmrdHeader:
+        """Create ISMRMRD header from KHeader.
+
+        All the parameters from the ISMRMRD header of the raw file used to create the kheader are saved in the
+        dictionary kheader['misc']. We first fill the information from this dictionary into the ISMRMRD header and then
+        we update the values based on the attributes of the KHeader.
+
+        Returns
+        -------
+            ISMRMRD header
+        """
+        header = ismrmrdschema.ismrmrdHeader()
+        header.version = self.misc['version']
+
+        def create_from_misc_dictionary(xsd_type, misc_dictionary_entry: dict):  # noqa: ANN001
```

Comment:
I would prefer not to do this .

Either we move these into our header, switch to using the ismrmrd header completely or be ok with loosing some of these on save.

The misc dictionary is a dumping ground for stuff that weigjt want to move into our header without any guarantees.
If some of these are so important or required for a valid ismrmd, we should move them into our header

Otherwise, This now makes.the dict it an untyped part of our API.
So it is the worst of all possible ways to deal with the header information...